LICENSES/Apache-2.0.txt

```diff
@@ -98,7 +98,7 @@
       (b) You must cause any modified files to carry prominent notices
           stating that You changed the files; and
 
-      (c) You must retain, in the Source form of any Derivative Works
+      You must retain, in the Source form of any Derivative Works
```

Comment:
I think this was probably overly-aggressive removal of '(c)' :-)

Reply:
🤦🏾

Reply:
Thanks for pointing this out.